shared/cfg/guest-os/Linux/RHEL/8/8.devel.cfg

```diff
@@ -0,0 +1,9 @@
+- devel:
+    # Uses defaults set in 8.cfg
+    # Adding x.y versions should only require changing the kickstart
+    # and adding per-arch image shas.
+    image_name = images/${os_variant}devel-${vm_arch_name}
+    unattended_install.cdrom:
+        cdrom_cd1 = isos/linux/RHEL8-devel-${vm_arch_name}.iso
```

Comment:
It used to be called `RHEL-7-devel-$arch.iso`, would you mind adjusting this to `RHEL-8-devel-${vm_arch_name}.iso` to comply?

Reply:
I have probably copied this from https://github.com/avocado-framework/avocado-vt/blob/f31b634ac555c7edaecc66f840cf30f6de8724c7/shared/cfg/guest-os/Linux/RHEL/7.devel/x86_64.cfg#L10
It should be corrected on both locations then.

Reply:
yes, please

Reply:
I made a PR #1950 , which corrects `cdrom_cd1` filenames.